from PyQt6 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(800, 600)
        self.centralwidget = QtWidgets.QWidget(parent=MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.tabWidget = QtWidgets.QTabWidget(parent=self.centralwidget)
        self.tabWidget.setGeometry(QtCore.QRect(120, 70, 501, 291))
        self.tabWidget.setObjectName("tabWidget")

        # Tab Khoản Chi
        self.tab = QtWidgets.QWidget()
        self.tab.setObjectName("tab")
        self.label = QtWidgets.QLabel(parent=self.tab)
        self.label.setGeometry(QtCore.QRect(10, 10, 71, 41))
        self.label.setObjectName("label")
        self.dateEdit = QtWidgets.QDateEdit(parent=self.tab)
        self.dateEdit.setGeometry(QtCore.QRect(140, 10, 151, 31))
        self.dateEdit.setObjectName("dateEdit")
        self.label_2 = QtWidgets.QLabel(parent=self.tab)
        self.label_2.setGeometry(QtCore.QRect(10, 50, 81, 41))
        self.label_2.setObjectName("label_2")
        self.lineEdit = QtWidgets.QLineEdit(parent=self.tab)
        self.lineEdit.setGeometry(QtCore.QRect(140, 60, 151, 22))
        self.lineEdit.setObjectName("lineEdit")
        self.lineEdit_2 = QtWidgets.QLineEdit(parent=self.tab)
        self.lineEdit_2.setGeometry(QtCore.QRect(140, 100, 151, 22))
        self.lineEdit_2.setObjectName("lineEdit_2")
        self.label_3 = QtWidgets.QLabel(parent=self.tab)
        self.label_3.setGeometry(QtCore.QRect(10, 90, 81, 41))
        self.label_3.setObjectName("label_3")
        self.comboBox = QtWidgets.QComboBox(parent=self.tab)
        self.comboBox.setGeometry(QtCore.QRect(140, 140, 151, 22))
        self.comboBox.setObjectName("comboBox")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.comboBox.addItem("")
        self.label_4 = QtWidgets.QLabel(parent=self.tab)
        self.label_4.setGeometry(QtCore.QRect(10, 130, 101, 41))
        self.label_4.setObjectName("label_4")
        self.pushButton = QtWidgets.QPushButton(parent=self.tab)
        self.pushButton.setGeometry(QtCore.QRect(120, 170, 111, 31))
        self.pushButton.setObjectName("pushButton")
        self.tabWidget.addTab(self.tab, "")

        # Tab Khoản Thu
        self.tab_2 = QtWidgets.QWidget()
        self.tab_2.setObjectName("tab_2")
        self.lineEdit_3 = QtWidgets.QLineEdit(parent=self.tab_2)
        self.lineEdit_3.setGeometry(QtCore.QRect(140, 60, 151, 22))
        self.lineEdit_3.setObjectName("lineEdit_3")
        self.dateEdit_2 = QtWidgets.QDateEdit(parent=self.tab_2)
        self.dateEdit_2.setGeometry(QtCore.QRect(140, 10, 151, 31))
        self.dateEdit_2.setObjectName("dateEdit_2")
        self.lineEdit_4 = QtWidgets.QLineEdit(parent=self.tab_2)
        self.lineEdit_4.setGeometry(QtCore.QRect(140, 100, 151, 22))
        self.lineEdit_4.setObjectName("lineEdit_4")
        self.pushButton_2 = QtWidgets.QPushButton(parent=self.tab_2)
        self.pushButton_2.setGeometry(QtCore.QRect(120, 170, 111, 31))
        self.pushButton_2.setObjectName("pushButton_2")
        self.label_5 = QtWidgets.QLabel(parent=self.tab_2)
        self.label_5.setGeometry(QtCore.QRect(10, 90, 81, 41))
        self.label_5.setObjectName("label_5")
        self.comboBox_2 = QtWidgets.QComboBox(parent=self.tab_2)
        self.comboBox_2.setGeometry(QtCore.QRect(140, 140, 151, 22))
        self.comboBox_2.setObjectName("comboBox_2")
        self.comboBox_2.addItem("")
        self.comboBox_2.addItem("")
        self.comboBox_2.addItem("")
        self.comboBox_2.addItem("")
        self.comboBox_2.addItem("")
        self.comboBox_2.addItem("")
        self.comboBox_2.addItem("")
        self.label_6 = QtWidgets.QLabel(parent=self.tab_2)
        self.label_6.setGeometry(QtCore.QRect(10, 130, 101, 41))
        self.label_6.setObjectName("label_6")
        self.label_7 = QtWidgets.QLabel(parent=self.tab_2)
        self.label_7.setGeometry(QtCore.QRect(10, 50, 81, 41))
        self.label_7.setObjectName("label_7")
        self.label_8 = QtWidgets.QLabel(parent=self.tab_2)
        self.label_8.setGeometry(QtCore.QRect(10, 10, 71, 41))
        self.label_8.setObjectName("label_8")
        self.tabWidget.addTab(self.tab_2, "")

        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(parent=MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 800, 26))
        self.menubar.setObjectName("menubar")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(parent=MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)

        self.retranslateUi(MainWindow)
        self.tabWidget.setCurrentIndex(1)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "Group"))
        self.label.setText(_translate("MainWindow",
                                      "<html><head/><body><p><span style=\" font-size:12pt;\">Ngày:</span></p></body></html>"))
        self.label_2.setText(_translate("MainWindow",
                                        "<html><head/><body><p><span style=\" font-size:12pt;\">Ghi chú:</span></p></body></html>"))
        self.label_3.setText(_translate("MainWindow",
                                        "<html><head/><body><p><span style=\" font-size:12pt;\">Tiền chi:</span></p></body></html>"))
        self.comboBox.setItemText(0, _translate("MainWindow", "Ăn uống"))
        self.comboBox.setItemText(1, _translate("MainWindow", "Chi tiêu hàng ngày"))
        self.comboBox.setItemText(2, _translate("MainWindow", "Quần áo"))
        self.comboBox.setItemText(3, _translate("MainWindow", "Mỹ phẩm"))
        self.comboBox.setItemText(4, _translate("MainWindow", "Y tế"))
        self.comboBox.setItemText(5, _translate("MainWindow", "Giáo dục"))
        self.comboBox.setItemText(6, _translate("MainWindow", "Đi lại"))
        self.label_4.setText(_translate("MainWindow",
                                        "<html><head/><body><p><span style=\" font-size:12pt;\">Danh mục:</span></p></body></html>"))
        self.pushButton.setText(_translate("MainWindow", "Thêm Khoản Chi"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab), _translate("MainWindow", "Khoản chi"))

        self.pushButton_2.setText(_translate("MainWindow", "Thêm Khoản Thu"))
        self.label_5.setText(_translate("MainWindow",
                                        "<html><head/><body><p><span style=\" font-size:12pt;\">Tiền thu:</span></p></body></html>"))
        self.comboBox_2.setItemText(0, _translate("MainWindow", "Ăn uống"))
        self.comboBox_2.setItemText(1, _translate("MainWindow", "Chi tiêu hàng ngày"))
        self.comboBox_2.setItemText(2, _translate("MainWindow", "Quần áo"))
        self.comboBox_2.setItemText(3, _translate("MainWindow", "Mỹ phẩm"))
        self.comboBox_2.setItemText(4, _translate("MainWindow", "Y tế"))
        self.comboBox_2.setItemText(5, _translate("MainWindow", "Giáo dục"))
        self.comboBox_2.setItemText(6, _translate("MainWindow", "Đi lại"))
        self.label_6.setText(_translate("MainWindow",
                                        "<html><head/><body><p><span style=\" font-size:12pt;\">Danh mục:</span></p></body></html>"))
        self.label_7.setText(_translate("MainWindow",
                                        "<html><head/><body><p><span style=\" font-size:12pt;\">Ghi chú:</span></p></body></html>"))
        self.label_8.setText(_translate("MainWindow",
                                        "<html><head/><body><p><span style=\" font-size:12pt;\">Ngày:</span></p></body></html>"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_2), _translate("MainWindow", "Khoản thu"))
